proto/cosmos/vesting/v1beta1/vesting.proto

```diff
@@ -83,3 +83,21 @@ message PermanentLockedAccount {
 
   BaseVestingAccount base_vesting_account = 1 [(gogoproto.embed) = true];
 }
+
+// TrueVestingAccount implements the VestingAccount interface. It provides
+// an account that can hold contributions subject to "lockup" (like a
+// PeriodicVestingAccount), or "true vesting" which is subject to clawback
+// of unvested tokens, or a combination (tokens vest, but are still locked).
+message TrueVestingAccount {
+  option (gogoproto.goproto_getters)  = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  BaseVestingAccount base_vesting_account = 1 [(gogoproto.embed) = true];
+
+  // funder_address specifies the account which can perform clawback.
+  string funder_address = 2;
```

Comment:
In this implementation we decided not to make it a privileged operation to create a vesting account, so it would be better to have a name that doesn't suggest a privileged role. Open to suggestions other than "funder", though.